test/engine-luatest/pagination_test.lua

```diff
@@ -1,7 +1,7 @@
 local server = require('test.luatest_helpers.server')
 local t = require('luatest')
 local tree_g = t.group('Tree index tests',
-    t.helpers.matrix{engine={'memtx'}, disable_ffi={true, false}})
+    t.helpers.matrix{engine={'memtx', 'vinyl'}, disable_ffi={true, false}})
```

Comment:
We don't need to disable ffi now (for `vinyl`, at least)

Reply:
Right. Removed vinyl+ffi from the test plan:

<details><summary>diff</summary>

```diff
diff --git a/test/engine-luatest/pagination_test.lua b/test/engine-luatest/pagination_test.lua
index 553d3a2380ba..46e12865f554 100644
--- a/test/engine-luatest/pagination_test.lua
+++ b/test/engine-luatest/pagination_test.lua
@@ -1,7 +1,11 @@
 local server = require('test.luatest_helpers.server')
 local t = require('luatest')
-local tree_g = t.group('Tree index tests',
-    t.helpers.matrix{engine={'memtx', 'vinyl'}, disable_ffi={true, false}})
+local tree_g = t.group('Tree index tests', {
+    {engine = 'memtx', disable_ffi = true},
+    {engine = 'memtx', disable_ffi = false},
+    -- Vinyl doesn't use FFI.
+    {engine = 'vinyl', disable_ffi = false},
+})
 
 -- TODO(gh-7739): Enable Lua JIT on aarch64.
 local function start_server(server)
```

</details>

Reply:
Nice. If I am the only reviewer, please hang `full-ci` label.